backend/src/errorHandler.ts

```diff
@@ -18,6 +19,8 @@ export default () => async (
       ctx.status = 400;
     } else if (error instanceof SettingMissingError) {
       ctx.status = 400;
+    } else if (error instanceof LackRequiredDataError) {
```

Comment:
could you please refactor this code section
try to add a new filed (statusCode) in Errors class, each Error could extend by one General Error

Reply:
Sure, I have refactored my implementation to fix that issue. And I will add my new request later.

Reply:
> Sure, I have refactored my implementation to fix that issue. And I have added my latest commit.

why just remove the code ?

Reply:
I found it is enough to use the existing `SettingMissingError`, so I chose to use it to throw the error instead of adding a new one. So I remove the `LackRequiredDataError` implementation.